composer.json

```diff
@@ -2,10 +2,13 @@
     "name": "carsonbot/carsonbot",
     "type": "project",
     "license": "MIT",
+    "repositories": [
+        { "type": "vcs", "url": "https://github.com/Nyholm/php-github-api"}
```

Comment:
I hope https://github.com/KnpLabs/php-github-api/pull/931 will be merged and released soon.

Reply:
@Nyholm merged and I've release v2.16.0!

Reply:
Awesome. 
Thank you for the quick merge and release.